### ISSUE TITLE
[Errno 126] Required key not available

### ISSUE THREAD

**Issue by f35f0ef9d0e827dae86552d3899f78fc** [borgbackup/borg#6571]
## Have you checked borgbackup docs, FAQ, and open Github issues?
Yes

## Is this a BUG / ISSUE report or a QUESTION?
QUESTION

#### Your borg version (borg -V).
1.2.0

#### Operating system (distribution) and version.
Arch Linux

#### Hardware / network configuration, and filesystems used.
EXT4 for backup path in question.

#### Full borg commandline that lead to the problem (leave away excludes and passwords)
```shell
borg create                        \
    --progress                      \
    --filter AME                     \
    --stats                             \
    --show-rc                        \
    --compression lz4           \
```

## Describe the problem you're observing.
I receive an error. Backup continues.

#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.
Yes. Encrypt a path using fscrypt. When borg backs up that path, it produces the error.

#### Include any warning/errors/backtraces from the system logs
>open: [Errno 126] Required key not available

## Question
Are the files encrypted by fscrypt backed up by borg, or are they skipped? Can I ignore the error, or am I expected to decrypt the files before backing up with borg?

**Comment by ThomasWaldmann** [comment]
The borg command you showed is incomplete.

The error message `open: [Errno 126] Required key not available` does not come from borg, I suspect it comes from `fscrypt` maybe?

**Comment by ThomasWaldmann** [comment]
In general, you can backup any files with borg, but of course borg needs to be able to discover them and read them.

**Comment by f35f0ef9d0e827dae86552d3899f78fc** [comment]
I looked into more based on your thoughts. Yeah, that error is from fscrypt.

I also reviewed the contents of my borg backup. The encrypted folder is 0 bytes and contains no files within the backup. I was hoping borg would back it up in like binary mode (?) so the encrypted file was still backed up. From what I can tell, the directory contents of the fscrypt folder are skipped. 😢 

For comparison's sake, I attempted to copy the encrypted folder from local machine to another path on local machine using rsync and I got essentially the same error:
>Required key not available (126)

So, I guess this is how fscrypt and the Linux kernel + filesystem work.
